### DISCUSSION TITLE
Add section to CONTRIBUTORS.rst or to wiki?

### DISCUSSION THREAD

**Discussion by Elliria** [autokey/autokey#803] [Maintenance goals]
Hey there,

I've been working on putting together beginner-friendly steps to make contributing to the AutoKey project as easy as possible. They're currently in [a GitHub gist](https://gist.github.com/Elliria/9ed92174d193ea8708cf1c8735749b31) in my account, but I'm thinking of either adding them to the bottom of the [CONTRIBUTORS.rst](https://github.com/autokey/autokey/blob/develop/CONTRIBUTORS.rst) file in the **develop** branch, but it occurs to me that it might be better to add them to the bottom of the [Contributing Code](https://github.com/autokey/autokey/wiki/Contributing-code) page in the wiki. Perhaps both?

**Comment by josephj11** [comment]
Either or both is fine with me.

My preference wiuld be to keep CONTRIBUTORS.rst short and add a link to the wiki.

Developers come at all levels of expertise, so doc heads should be like man pages, short and to the point with links to supporting information. Also, we're probably going to be learning about the development process for a long time and it will be much easier to keep the wiki up to date with no need for PRs...

**Reply by Elliria** [reply]
Okay, I did that. There's now a small [Help is welcome](https://github.com/autokey/autokey/wiki/Contributing-Code#help-is-welcome) section in the [Contributing Code](https://github.com/autokey/autokey/wiki/Contributing-Code) page in the wiki and there's also the new [GitHub Cheat Sheet](https://github.com/autokey/autokey/wiki/GitHub-Cheat-Sheet) page in the wiki. I've also gotten rid of my two gists that contained that information since it would be easier to just maintain them in one place and they were written specifically for the AutoKey project, so they shouldn't "live" with me.

**Comment by josephj11** [comment]
I will have to read all of that later, but the organization is good.
